### DISCUSSION TITLE
NavigationView header binding

### DISCUSSION THREAD

**Discussion by Kattabomane** [lepoco/wpfui#896] [Q&A]
Hello,
I am trying to use the latest preview (3.0-preview-13) of WPF UI.
When using NavigationView, how to change header according to selected page ?
ie how to bind the title according to selected page ?
Thanks for your help
K.

**Comment by Djohnnie** [comment]
Your page itself can have a property ui:NavigationView.HeaderContent where you can specify the header text that will be displayed on top in the navigation view.
